### PR TITLE
I18n sync Down & Out 04/30

### PR DESCRIPTION
**Pull request by deploy-code-org**

### CODE REVIEW

File: i18n/locales/es-MX/dashboard/blocks.yml [hunk 1]
```diff
@@ -553,7 +553,7 @@ es-MX:
             '"ROBOT"': todos los robots
             '"SHARK"': todos los tiburones
       Dancelab_setTintInline:
-        text: definir tinte [SPRITE] [0] a [VAL] [1]
+        text: definir tinte {SPRITE} a {VAL}
```
Comment (bethanyaconnor) [inline i18n/locales/es-MX/dashboard/blocks.yml:556]: hmm this doesn't look right but it's also not what's in [crowdin](https://crowdin.com/translate/codeorg/639/enus-esmx#q=Dancelab_setTintInline)

Comment (Hamms) [reply i18n/locales/es-MX/dashboard/blocks.yml:556]: No, this is right. The stuff that's on crowdin is the redacted version; things like `[SPRITE] [0] a [VAL] [1]` are what show up when the translator adds spaces. This change is reflective of the errant spaces being removed from the translation, so the restoration process works again.

Comment (bethanyaconnor) [reply i18n/locales/es-MX/dashboard/blocks.yml:556]: Great thanks for the explanation!

**Review by Hamms — APPROVED** [review]

**Comment by bethanyaconnor** [comment]
@Hamms @nkiruka PR triage was concerned with any risk for non-English speakers. Can someone own merging this on Friday as I'll be out?

**Review by nkiruka — APPROVED** [review]

**Pull request merged**